odxtools/diagcodedtypes.py

```diff
@@ -478,11 +493,28 @@ def __init__(
             base_type_encoding=base_type_encoding,
             is_highlow_byte_order_raw=is_highlow_byte_order_raw,
         )
-        self.length_key_id = length_key_id
+        self.length_key_ref = length_key_ref
```

Comment:
small breaking change

Reply:
in principle its a breaking change, but I'm will bet a box of Champaign that nobody outside of this repository ever used this attribute...